app/services/project_job_service.rb

```diff
@@ -7,6 +7,9 @@ def initialize(project:)
 
   def list_contents_job(user:)
     job = ListProjectContentsJob.perform_later(user_id: user.id, project_id: @project.id)
+    # Log the job id and the Sidekiq JID in case we need to troubleshoot the job
+    # https://github.com/sidekiq/sidekiq/wiki/Active-Job#job-id
+    Rails.logger.info("Job scheduled, job id: #{job.job_id}, (Sidekiq JID: #{job.provider_job_id || 'nil'})")
```

Comment:
This could be handy in the future so we can know for sure the state of our jobs in Sidekiq. For example we could query Sidekiq to see the status (enqueued, in process, dead). This will require that we save the `JID` value to the database and use the Sidekiq API to query for its status, which is not part of this PR, but something we could do in the future.